README.md

```diff
@@ -32,9 +31,7 @@ need to be added to the PATH environment variable.
 
 ## Authentication requirements - specific for each service backend
 
-- AWS: requires the access key/secret access key pair
 - Azure: requires an AD application/service principal
```

Comment:
We should reword this to remove references to "each service backend". It doesn't make sense to have a single bullet point when we only support Azure.